monocle/web_utils.py

```diff
@@ -138,10 +138,12 @@ def get_spawnpoint_markers():
         spawns = session.query(Spawnpoint)
         return [{
             'spawn_id': spawn.spawn_id,
-            'despawn_time': spawn.despawn_time,
+            'despawn_time': spawn.despawn_time
+                if spawn.updated and spawn.updated > conf.LAST_MIGRATION else None,
```

Comment:
after removing the `spawn.updated and ` I get `TypeError: unorderable types: NoneType() > int()`
guess I'll have to readd it